env/env.cc

```diff
@@ -597,18 +606,29 @@ Status Env::LoadEnv(const std::string& value, Env** result) {
 
 Status Env::LoadEnv(const std::string& value, Env** result,
                     std::shared_ptr<Env>* guard) {
+  return CreateFromString(ConfigOptions(), value, result, guard);
+}
+
+Status Env::CreateFromString(const ConfigOptions& config_options,
+                             const std::string& value, Env** result,
+                             std::shared_ptr<Env>* guard) {
   assert(result);
+  if (value.empty()) {
+    *result = Env::Default();
+    return Status::OK();
+  }
   Status s;
 #ifndef ROCKSDB_LITE
   Env* env = nullptr;
   std::unique_ptr<Env> uniq_guard;
   std::string err_msg;
   assert(guard != nullptr);
+  (void)config_options;
   env = ObjectRegistry::NewInstance()->NewObject<Env>(value, &uniq_guard,
                                                       &err_msg);
   if (!env) {
-    s = Status::NotFound(std::string("Cannot load ") + Env::Type() + ": " +
-                         value);
+    s = Status::NotSupported(std::string("Cannot load ") + Env::Type() + ": " +
```

Comment:
Do we need to change the returned status from NotFound to NotSpported？Or you want to align with the other classes' CreateFromString return status?

Reply:
I wanted to make it align with the other methods that return NotSupported if an instance could not be created.